terraform/deploy/variables.tf

```diff
@@ -15,11 +15,11 @@ variable "cognito" {
   type        = map(string)
 
   default = {
-    name                        = "cognito"
-    issuer                      = "https://cognito-idp.eu-west-2.amazonaws.com/user_pool_id"
-    client_id_ssm_parameter     = "/path/to/ssm_param"
-    client_secret_ssm_parameter = "/path/to/ssm_param"
-    admin_group                 = "ci_admin"
+    name          = "concourse"
```

Comment:
This block looks stale to me. This isn't the config we're running, so can this lot just be removed now?

Reply:
yep lots of that I think